src/doubleTrouble.js

```diff
@@ -24,7 +24,13 @@
  * @return {number[]} - array of remained numbers
  */
 function doubleTrouble(numbers, target) {
-  // write code here
+  return numbers.reduce((acc, item) => {
+    if (acc[acc.length - 1] + item !== target) {
```

Comment:
what is `acc`?

Reply:
this is accumulator of values. Renamed to resultValues